src/components/common/ColonyActions/helpers/mapItemToMessageFormat.tsx

```diff
@@ -74,12 +75,17 @@ export const mapColonyActionToExpectedFormat = (actionData: ColonyAction) => {
       actionData.toDomain?.metadata?.name ??
       formatMessage({ id: 'unknownDomain' }),
     tokenSymbol: actionData.token?.symbol,
-    // reputationChangeNumeral: item.reputationChange && (
-    //   <Numeral value={item.reputationChange} decimals={Number(item.decimals)} />
+    reputationChangeNumeral: actionData.amount && (
+      <Numeral
+        value={actionData.amount}
+        // decimals={actionData.token?.decimals ?? DEFAULT_TOKEN_DECIMALS}
```

Comment:
I'm not Raul, but If I remember correctly from my time implementing the reputation actions... The recommended way to earn rep is to be paid by the native token of the colony, when you pay someone with the native token they earn reputation 1:1. That's why the `decimals` of the native token would be relevant if you want to format reputation amounts, for example.

Reply:
@jakubcolony I left the comment here because I had trouble getting the formatting correct when using the decimals so wanted to confirm it was necessary.

Reply:
For rep, decimals come from the native token, as Armando pointed out.

As for storing, yes, since we already have them. In this case you would use the decimals for the colony's native token, which is available in the `colony` object